features/02_configure_aruba/command_runtime_environment.feature

```diff
@@ -19,6 +19,7 @@ Feature: Define default process environment
     Given a file named "spec/environment_spec.rb" with:
     """ruby
     require 'spec_helper'
+    require 'aruba/rspec'
```

Comment:
Why did you add this? Do you see an error? That's should already provided by the fixture in `spec/support/aruba.rb`.